tensorflow/contrib/rnn/kernels/BUILD

```diff
@@ -0,0 +1,19 @@
+# Libraries and Kernels for RNNs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_kernel_library(
+    name = "lstm_ops",
+    prefix = "lstm",
+    deps = [
+        "//tensorflow/contrib/rnn:lstm_ops_op_lib",
+        "//tensorflow/core:framework",
```

Comment:
A `tf_cusom_op_library` cannot depend (even transitively) on `core:framework`. It should depend on `core:framework_headers_lib` instead. I'll update the doc to reflect that. Meanwhile, try replacing this.